library/src/skins/default/src/sass/modules/tool/samigo/_samigo.scss

```diff
@@ -466,6 +466,10 @@
 	sakai-user-photo {
 		margin-right: 7px;
 	}
+
+	.portletBody {
+	    margin-bottom: 3rem !important;
+	}
```

Comment:
I believe we should utilize the class defined in `modules/tool/_base.scss`. The class already exists; we simply need to incorporate the new margin.

Reply:
@kunaljaykam - Thanks. My initial intent was to isolate this change solely to T&Q. Upon looking at views in other tools (e.g., Discussions) it looks like the margin-bottom would be welcome there as well. I'll force push a revision imminently.